advanced_usage/drain_mode.md

```diff
@@ -0,0 +1,37 @@
+# **Drain Mode**
+
+**Note: **Using this feature requires atleast version `19.1.0`  of the GoCD server.
+
+The GoCD system includes a lot of internal subsystems and processes, which continuously do work. 
+Some examples include the material subsystem which takes care of polling for materials, the scheduling subsystem which automatically starts pipelines and stages, the agent subsystem which continuously keeps tracks of agent states and provides them jobs to run, etc.
+
+The drain mode is a maintenance mode which a GoCD system administrator can put GoCD into so that it is safe to restart it or upgrade it without having running jobs reschedule when it is back.
```

Comment:
I think we might have to rephrase this to something like 

"Drain mode is a maintenance mode which can be utilized by the GoCD system administrator to safely restart or upgrade the GoCD server without having any running jobs reschedule when the server is back up."